Modules/IO/TIFF/src/itkTIFFImageIO.cxx

```diff
@@ -148,8 +148,8 @@ TIFFImageIO::GetFormat()
 void
 TIFFImageIO::ReadVolume(void * buffer)
 {
-  const int width = m_InternalImage->m_Width;
-  const int height = m_InternalImage->m_Height;
+  const size_t width = m_InternalImage->m_Width;
+  const size_t height = m_InternalImage->m_Height;
 
```

Comment:
I agree with you that `width` and `height` should not be signed int, but I think they should be `auto`, or `uint32_t` (not `size_t`), in order to pass them safely to `TIFFReadRGBAImageOriented` (a few lines later). Please check!

Reply:
In ReadVolume function, they are just used by  ReadCurrentPage function, which expect a size_t.

Reply:
Oops, I see now, the `TIFFReadRGBAImageOriented(m_InternalImage->m_Image, width, height...)` call is in another member function.  Sorry for the noise!

Reply:
To ensure that no narrowing (signed-to-unsigned or otherwise) conversion takes place, you could use C++11 curly-braced list-initialization:

    const size_t width{ m_InternalImage->m_Width };
    const size_t height{ m_InternalImage->m_Height };

I think that's preferable, especially because the size of `size_t` is platform dependent. What do you think?

Reply:
I'll do that